Assets/_Laz/Scripts/Environment/ElectricGate.cs

```diff
@@ -0,0 +1,77 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Laz
+{
+    [RequireComponent(typeof(BoxCollider))]
+    public class ElectricGate : MonoBehaviour
```

Comment:
Can you change the class to be called ElectricGateBehaviour? Since it's deriving from MonoBehaviour